source/NeutronaWand/Actions.h

```diff
@@ -485,8 +485,8 @@ void checkWandAction() {
 
         // Menu Level 1: Intensify: Cycle through the modes (Video Game, Cross The Streams, Cross The Streams Mix)
         // Menu Level 1: Barrel Wing Button: Enable Spectral and Holiday modes.
-        // Menu Level 2: Intensify: Vent Light Auto Intensity.
-        // Menu Level 2: Barrel Wing Button: 5 / 48 / 60 barrel LEDs.
+        // Menu Level 2: Intensify: Enable pack vibration, enable pack vibration while firing only, disable pack vibration, reset to defaults. *Note that the pack vibration switch will toggle both pack and wand vibration on or off*
```

Comment:
Does this mean you must enable some kind of pack vibration to also get wand vibration?

Reply:
No; it means you can have separate settings for the Pack and Wand.